docs/guides/distribution/publishing.md

```diff
@@ -20,6 +20,10 @@ Every framework has its publishing tooling. It is outside of the scope of this d
 
 <Command name="build" />
 
-This command embeds your web assets into a single binary with your Rust code. The binary itself will be located in `src-tauri/target/release/[app name]`, and installers will be located in `src-tauri/target/release/bundle/`.
+This command embeds your web assets into a single binary with your Rust code. The binary itself will be located in `src-tauri/target/release/[os name]`, and installers will be located in `src-tauri/target/release/bundle/`.
+
+For example, for MacOS:
+- The runnable binary will be `src-tauri/target/release/bundle/macos/[app name].app`
+- The installer will be `src-tauri/target/release/bundle/dmg/[app name].dmg`
```

Comment:
based on @FabianLars's comment (https://github.com/tauri-apps/tauri-docs/issues/2692#issuecomment-2352313477), i'll replace "installers" with "bundles and installers".  if i understood correctly: 
```suggestion
This command embeds your web assets into a single binary with your Rust code. The binary itself will be located in `src-tauri/target/release/[app name]`, and bundles and installers will be located in `src-tauri/target/release/bundle/`.
```